daras_ai_v2/gpu_server.py

```
@@ -63,26 +61,17 @@ def call_celery_task_outfile_with_ret(
+    if not num_outputs:
+        return [], {}
     with ExitStack() as stack:
-        for blob in blobs:
-            stack.enter_context(
-                register_blob(blob, filename=filename, content_type=content_type)
-            )
+        generatd_urls = [
+            stack.enter_context(generate_signed_url(filename, content_type))
+            for _ in range(num_outputs)
+        ]
+        pipeline["upload_urls"], public_urls = zip(*generatd_urls)
```

Ruff (0.15.17)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)